docs/release/trg-0/trg-0.md

```diff
@@ -4,13 +4,14 @@ title: TRG 0 - Changelog & Drafts
 
 ## Index of Tractus-X Release Guidelines (TRGs)
 
-| Created      | Post-History                                                                                  |
-|--------------|-----------------------------------------------------------------------------------------------|
-| 8-May-2023   | Move TRG 7.00-7.07 out of Draft to Final
-| 7-Mar-2023   | Remove author column, reorder post-history latest first, cleanup formatttings across all trgs |
-| 7-Mar-2023   | Move TRG 2.05, 5.09, 5.10 and 5.11 out of Draft to Prerelease                                 |
-| 23-Feb-2023  | Add draft "Helm Action"                                                                       |
-| 13-Sept-2022 | Initial contribution                                                                          |
+| Created      | Post-History                                                                                                           |
+|--------------|------------------------------------------------------------------------------------------------------------------------|
+| 9-May-2023   | Better describe base images (4.02), Make DockerHub registry mandatory (4.05), Describe notice for Docker images (4.06) |
```

Comment:
I would potentially only add the critical things which everyone needs to do (in sense of new trg's) or put the most critical things first. But yeah we can also remakr small changes.

Reply:
Ok good remark, i slightly adapted it in https://github.com/eclipse-tractusx/eclipse-tractusx.github.io/pull/216/commits/58bce3399d499815b78312fbb4ea03b53e97825c. 
Will merge with this change, can still improve afterwards if necessary